### PR TITLE
enhance: fetch simdjson directly in the format of targz

### PR DESCRIPTION
**Pull request by yellow-shine**

### CODE REVIEW

File: internal/core/thirdparty/simdjson/CMakeLists.txt [hunk 1]
```diff
@@ -13,7 +13,6 @@
 
 FetchContent_Declare(
     simdjson
-    GIT_REPOSITORY https://github.com/simdjson/simdjson.git
-    GIT_TAG        v3.1.7
+    URL https://github.com/simdjson/simdjson/archive/refs/tags/v3.1.7.tar.gz
```
Comment (XuanYang-cn) [inline internal/core/thirdparty/simdjson/CMakeLists.txt:16]: Please added URL_HASH MD5=1b0d75ad32179c77f84f4a09d4214057

or it'll try to download everytime.

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: internal/core/thirdparty/simdjson/CMakeLists.txt (hunk 1 above)
Comment (XuanYang-cn) [reply internal/core/thirdparty/simdjson/CMakeLists.txt:16]: ref: https://cmake.org/cmake/help/latest/module/FetchContent.html
image

**Comment by yanliang567** [comment]
manual pass for build success

**Pull request merged**